packages/bitcoin-esplora-api-provider/lib/BitcoinEsploraApiProvider.js

```diff
@@ -134,13 +134,18 @@ export default class BitcoinEsploraApiProvider extends Provider {
 
     const confirmations = tx.status.confirmed ? currentHeight - tx.status.block_height + 1 : 0
 
+    const totalFee = tx.fee
+    const fee = Math.round(totalFee / Math.ceil(tx.weight / 4))
```

Comment:
Can we use `BigNumber.js` here? Division operation may create issues.

Reply:
Doing rounding with bignumber is super annoying
and this shouldn't really cause an issue like this? value is in sats (well below max int limits) and since every value computed is intended for rounding anyway (.round, and .ceil), precision should not affect it